plugins/nf-google/build.gradle

```diff
@@ -40,7 +40,10 @@ dependencies {
     api 'com.google.apis:google-api-services-lifesciences:v2beta-rev20210527-1.31.5'
     api 'com.google.auth:google-auth-library-oauth2-http:0.18.0'
     api 'com.google.cloud:google-cloud-batch:0.29.0'
-    api 'com.google.cloud:google-cloud-logging:3.8.0'
+    api ('com.google.cloud:google-cloud-logging:3.8.0') {
+        // use version of bouncycastle provided by nextflow
+        exclude group: 'org.bouncycastle'
```

Comment:
think a bit more about this. I'm tempted to suggest to add an explicit dependency so that plugin continue to use its own copy

Reply:
Checking more about this I've noticed google-cloud-logging:3.15.0 and later is not using anymore bouncycastle. It would make more sense to upgrade the dependency instead of pinning the version. 

Same for google-cloud-batch requiring protobuf

Reply:
Updated versions of `google-cloud-logging` and `google-cloud-batch`.

Reply:
nice, let's how tests go